app/services/acceptable_verified_credential_adoption_service.rb

```diff
@@ -8,12 +8,19 @@
 class AcceptableVerifiedCredentialAdoptionService
   attr_accessor :user
 
+  STATS_KEY = 'api.user_transition_availability'
+
   def initialize(user)
     @user = user
   end
 
   def perform
     display_organic_modal_for_logingov_conversion
+    send_email_for_reactivation
```

Comment:
I'm kinda confused on exactly what is going on here. What is the point of having `reactivation_email:` in a response to show a modal or not?

Reply:
Initially I was trying to make this service as easy as possible to use on the front end, but seeing as how the email will not really need front end interaction you're right. Better to separate the concerns. Thanks for the feedback! I'll provide a separate path for the reactivation portion of avc adoption.

Reply:
@rileyanderson When you have a moment, would you mind taking a second look and letting me know if this approach makes sense? Happy to chat on slack if you have any questions. Thanks!